code401challenges/src/test/java/linkedList/LinkedListTest.java

```diff
@@ -0,0 +1,46 @@
+package linkedList;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+
```

Comment:
these tests do not cover all of the tests listed in the assignment: 

- [ ] Can successfully instantiate an empty linked list
- [x] Can properly insert into the linked list
- [x] The head property will properly point to the first node in the linked list
- [x] Can properly insert multiple nodes into the linked list
- [ ] Will return true when finding a value within the linked list that exists
- [x] Will return false when searching for a value in the linked list that does not exist
- [x] Can properly return a collection of all the values that exist in the linked list